docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -100,8 +100,9 @@ There are two ways to configure the trace to logs feature:
    You can also click **Open advanced data source picker** to see more options, including adding a data source.
 
 1. Set start and end time shift. As the logs timestamps may not exactly match the timestamps of the spans in trace it may be necessary to search in larger or shifted time range to find the desired logs.
-1. Select which tags to use in the logs query. The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful, for example, if the tag has dots in the name and the target data source does not allow using dots in labels. In that case, you can for example remap `http.status` (the span attribute) to `http_status` (the data source field). "Data source" in this context can refer to Loki, or another log data source.
-1. Optionally switch on the **Filter by trace ID** and/or **Filter by span ID** setting to further filter the logs if your logs consistently contain trace or span IDs.
+1. Select which tags to use in the logs query.
+   The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful, for example, if the tag has dots in the name and the target data source does not allow using dots in labels. In that case, you can for example remap `http.status` (the span attribute) to `http_status` (the data source field). "Data source" in this context can refer to Loki, or another log data source.
+1. Optional: Switch on the **Filter by trace ID** and/or **Filter by span ID** setting to further filter the logs if your logs consistently contain trace or span IDs.
```

Comment:
https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanaandor

```suggestion
1. Optional: Switch on the **Filter by trace ID** and **Filter by span ID** settings to further filter the logs if your logs consistently contain trace or span IDs.
```

Reply:
Okay, but. I have some disagreement with the linting rule on this. In procedures, using "and" has an implicit instruction to select all options listed. Using "and/or", while not pretty, is a succinct way of saying "You can use either or both of these options:". I need to go back into the Traces to logs part to verify the user's action to see how to rewrite this sentence. I am not sure changing it to "and" is correct.